rustls/src/msgs/handshake_test.rs

```diff
@@ -779,7 +779,7 @@ fn sample_client_hello_payload() -> ClientHelloPayload {
             cookie: Some(PayloadU16::new(vec![1, 2, 3])),
             signature_schemes: Some(vec![SignatureScheme::ECDSA_NISTP256_SHA256]),
             session_ticket: Some(ClientSessionTicket::Request),
-            ec_point_formats: Some(ECPointFormat::SUPPORTED.to_vec()),
+            ec_point_formats: Some(SupportedEcPointFormats { uncompressed: true }),
```

Comment:
Maybe implement `Default` for `SupportedEcPointFormats`?

Reply:
removed the derive and impl'd with `uncompressed: true`